lizmap/www/js/attributeTable.js

```diff
@@ -3147,12 +3148,23 @@ var lizAttributeTable = function() {
             && config.layers[lname]['geometryType'] != 'none'
             && config.layers[lname]['geometryType'] != 'unknown') {
             var lConfig = config.layers[lname];
-            selectionLayersSorted[config.attributeLayers[lname].order] = '<option value="'+lname+'">'+lConfig.title+'</option>';        }
+            var selectionOption = '<option value="'+lname+'">'+lConfig.title+'</option>';
+
+            if(lname in config.attributeLayers){
+                selectionLayersSorted[config.attributeLayers[lname].order] = selectionOption;
+            }else{
+                selectionLayersNotSorted.push(selectionOption);
+            }
+        }
     });
 
+    // We put selection layers which are also set in attribute table tool first then others
     for (var i = 0; i < selectionLayersSorted.length; i++) {
         options += selectionLayersSorted[i];
     }
+    for (var i = 0; i < selectionLayersNotSorted.length; i++) {
+        options += selectionLayersNotSorted[i];
+    }
```

Comment:
Can you remove these 3 lines ? Selection can't be yet performed without attribute table configuration

Reply:
@rldhont Do you still want those lines to be removed ?

Reply:
@nboisteault I have commented these lines before merging.